packages/babel-parser/test/plugin-options.js

```diff
@@ -14,9 +14,10 @@ describe("plugin options", function() {
     const OPT_2 = [NAME, { decoratorsBeforeExport: false }];
     const SYNTAX_1 = "@dec export class C {}";
     const SYNTAX_2 = "export @dec class C {}";
-    const SYNTAX_DEFAULT = "export @dec class C {}";
+    const SYNTAX_DEFAULT = "";
 
-    it("when they aren't specified", function() {
+    // Currently every option is required, so this can't be tested.
+    it.skip("when they aren't specified", function() {
```

Comment:
Maybe we can just remove this dead code?

Reply:
I forgot about flow's `all` option :laughing: I re-enabled the test using it.